tools/gen_addon_banner.py

```diff
@@ -0,0 +1,85 @@
+# License AGPLv3 (https://www.gnu.org/licenses/agpl-3.0-standalone.html)
+# Copyright (c) 2018 ACSONE SA/NV
+# Copyright (c) 2018 GRAP (http://www.grap.coop)
+
+import os
+import click
+from .manifest import read_manifest, find_addons, NoManifestFound
+from .gitutils import commit_if_needed
+from PIL import Image
+
+
+def gen_one_addon_banner(addon_name, addon_dir, manifest, background, font):
+    # It would be cleaner to modify the readme, but it is easier
+    image = Image.open(background)
+    # TODO:Maybe we could add the title of the addon...
+    modified = []
+    for path in manifest["images"]:
+        image.save("%s/%s" % (addon_dir, path))
```

Comment:
> Does it really make sense to save one copy of this banner in every OCA module? Isn't it easier just to modify the README template (contained in this repo too) to include it from this maintainer-tools github repo directly?
> 
> Even more taking into account [OCA/oca-addons-repo-template#190 (comment)](https://github.com/OCA/oca-addons-repo-template/pull/190#issuecomment-1824499220).

@yajo I think we should be able to put custom banners for modules. Same as icons.

Reply:
OK, but if it isn't custom, do we really need to replicate this image thousands of times?

For the icon, we do need it, because Odoo expects the icon to exist in a specific place. But the banner only appears on the compiled readme/index